cms-adapter/src/Rsbc.Dmf.CaseManagement/Manager/CaseManager.cs

```diff
@@ -3775,9 +3859,10 @@ private static async Task<IEnumerable<incident>> SearchCases(DynamicsContext ctx
                 .Expand(i => i.dfp_ClinicId)
                 .Expand(i => i.dfp_MedicalPractitionerId)
                 .Expand(i => i.bcgov_incident_bcgov_documenturl)
+                .Expand(i => i.dfp_incident_dfp_knownmedicalcondition)
```

Comment:
note to self: another method that does not appear to be used, add breakpoint here and see if we can remove these changes and mark this method "Obsolete"